run.sh

```diff
@@ -36,6 +44,7 @@ yarn global add package-json-merge > /dev/null 2>&1
 package-json-merge package.json packageB.json > packageA.json
 mv packageA.json package.json
 rm packageB.json
+git init
```

Comment:
I'd add a TODO comment for repository integration

Reply:
There will be no need :wink: